c2cgeoform/scaffolds/c2cgeoform/+package+/models/c2cgeoform_demo.py_tmpl

```diff
@@ -310,6 +310,7 @@ class Excavation(Base):
                 'typ': colander_ext.Geometry(
                     'MULTIPOLYGON', srid=4326, map_srid=21781),
                 'widget': deform_ext.MapWidget(
+                    projection='EPSG:21781',
```

Comment:
Not possible to use the new projection (2056) ?

Reply:
Yes. I would have a look at this soon.